src/components/post/HtmlRenderer.tsx

```
@@ -0,0 +1,86 @@
+              max-width: 100%;
+            }
+          </style>
+          </head>
+          <body>
+            ${content}
+          </body>
+          </html>
+        `;
+  };
+
+  return <div dangerouslySetInnerHTML={{__html: returnHtmlContent()}} />;
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)